advanced/Scripts/updatecheck.sh

```diff
@@ -41,54 +41,78 @@ function get_local_version() {
 # shellcheck disable=SC1091
 . /etc/pihole/setupVars.conf
 
+utilsfile="/opt/pihole/utils.sh"
+source "${utilsfile}"
+
+# Remove the below three legacy files if they exist
+GITHUB_VERSION_FILE="/etc/pihole/GitHubVersions"
+LOCAL_BRANCH_FILE="/etc/pihole/localbranches"
+LOCAL_VERSION_FILE="/etc/pihole/localversions"
+
+if [ -f "${GITHUB_VERSION_FILE}" ]; then
+    rm "${GITHUB_VERSION_FILE}"
+fi
+
+if [ -f "${LOCAL_BRANCH_FILE}" ]; then
+    rm "${LOCAL_BRANCH_FILE}"
+fi
+
+if [ -f "${LOCAL_VERSION_FILE}" ]; then
+    rm "${LOCAL_VERSION_FILE}"
+fi
```

Comment:
You can use `rm -f <filename>` instead and the file will be deleted if found and`rm` will not error out if the file doesn't exist.

I'd also just directly `rm` the file instead of the variable since the file locations are in a known path and couldn't be located anywhere else.